commit 158de31ec7d4478e809d0f770f3e8d6e9a050606
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Mon Sep 24 16:57:09 2018 +0100

    staging: mmal-vchiq: Make timeout a defined parameter
    
    The timeout period for VPU communications is a useful thing
    to extend when debugging.
    Set it via a define, rather than a magic number buried in the code.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/staging/vc04_services/vchiq-mmal/mmal-vchiq.c b/drivers/staging/vc04_services/vchiq-mmal/mmal-vchiq.c
index 4bc6b87881a2..efed0b68bbe0 100644
--- a/drivers/staging/vc04_services/vchiq-mmal/mmal-vchiq.c
+++ b/drivers/staging/vc04_services/vchiq-mmal/mmal-vchiq.c
@@ -44,6 +44,12 @@ MODULE_VERSION("0.0.1");
  */
 #define VCHIQ_MMAL_MAX_COMPONENTS 64
 
+/*
+ * Timeout for synchronous msg responses in seconds.
+ * Helpful to increase this if stopping in the VPU debugger.
+ */
+#define SYNC_MSG_TIMEOUT	3
+
 /*#define FULL_MSG_DUMP 1*/
 
 #ifdef DEBUG
@@ -691,7 +697,7 @@ static int send_synchronous_mmal_msg(struct vchiq_mmal_instance *instance,
 	}
 
 	timeout = wait_for_completion_timeout(&msg_context->u.sync.cmplt,
-					      3 * HZ);
+					      SYNC_MSG_TIMEOUT * HZ);
 	if (timeout == 0) {
 		pr_err("timed out waiting for sync completion\n");
 		ret = -ETIME;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1740_158de31ec7d4478e809d0f770f3e8d6e9a050606.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1740_158de31ec7d4478e809d0f770f3e8d6e9a050606.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
